.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+### Prerequisites
+
+- [ ] I have written a descriptive issue title
+- [ ] I have verified that I am running the latest version of Nancy
+- [ ] I have verified if the problem exist in both `DEBUG` and `RELEASE` mode
+- [ ] I have searched [open issues](https://github.com/NancyFx/Nancy/issues) to ensure it has not already been reported

```

Comment:
Would it be better to say open and closed issues? I saw the [angular-ui team](https://github.com/angular-ui/bootstrap/blob/master/.github/ISSUE_TEMPLATE.md) mentioned both since the issue could have been resolved already.